src/main/java/frc/robot/subsystems/tank/TankSubsystem.java

```diff
@@ -52,8 +60,7 @@ public TankSubsystem() {
   public void periodic() {
     // This method will be called once per scheduler run
     // TODO: odometry
-    leftMain.set(ControlMode.PercentOutput, yScale);
-    rightMain.set(ControlMode.PercentOutput, angularScale);
+    differentialDrive.tankDrive(yScale, angularScale, squareInput);
```

Comment:
I looked at docs and it seems like DifferentialDrive lacks methods to change the ControlMode, unlike TalonSRX, where you can specify the control mode when calling .set(). Setting motor power directly using TalonSRX has the benefit of being able to choose PercentOutput, Velocity, Position, or Current. (http://www.ctr-electronics.com/downloads/api/java/html/classcom_1_1ctre_1_1phoenix_1_1motorcontrol_1_1can_1_1_talon_s_r_x.html#aa8a9315c026864fb9754bd3379a38871)

Reply:
This is a really good point, and worth having in the main discussion. Another consideration is which library (yours or WPI) is better tested and supported?